Features/Dialogue/DialogueAnimation.cs

```diff
@@ -1,14 +1,16 @@
 using Godot;
 using untitledplantgame.Common;
+using DialogueLine = untitledplantgame.Dialogue.Models.DialogueLine;
 
 namespace untitledplantgame.Dialogue;
 
 public partial class DialogueAnimation : Node
 {
-	private const float CharacterPerSecond = 40; // range 25 - 40 
+	private const float CharacterPerSecond = 40; // range 25 - 40
 	public bool IsPlaying { get; private set; }
 
-	[Export] private Timer _timer;
+	[Export]
+	private Timer _timer;
 	private Logger _logger;
```

Comment:
Nitpicky, but should be after the public attributes

Reply:
adjusted